src/test/java/io/quarkus/search/app/indexing/SchedulerTest.java

```diff
@@ -0,0 +1,68 @@
+package io.quarkus.search.app.indexing;
+
+import static io.restassured.RestAssured.given;
+import static io.restassured.RestAssured.when;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.Duration;
+import java.util.Map;
+
+import io.quarkus.search.app.SearchService;
+import io.quarkus.search.app.dto.GuideSearchHit;
+import io.quarkus.search.app.dto.SearchResult;
+import io.quarkus.search.app.testsupport.QuarkusIOSample;
+
+import io.quarkus.test.common.http.TestHTTPEndpoint;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.QuarkusTestProfile;
+import io.quarkus.test.junit.TestProfile;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+import org.awaitility.Awaitility;
+
+import io.restassured.common.mapper.TypeRef;
+
+@QuarkusTest
+@TestProfile(SchedulerTest.Profile.class)
+@TestHTTPEndpoint(SearchService.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@QuarkusIOSample.Setup(filter = QuarkusIOSample.SearchServiceFilterDefinition.class)
+class SchedulerTest {
+    public static class Profile implements QuarkusTestProfile {
+        @Override
+        public Map<String, String> getConfigOverrides() {
+            return Map.of("indexing.on-startup.enabled", "false",
+                    "indexing.scheduled.enabled", "true",
+                    // every 1 minute:
+                    "indexing.scheduled.cron", "0 0/1 * ? * * *");
```

Comment:
Would something lower (e.g. 5s, or 15s) make sense?
I'd rather limit the impact on the duration of the test suite...